pages/linux/kill.md

```diff
@@ -10,7 +10,7 @@
 
 - List signal values and their corresponding names (to be used without the `SIG` prefix):
 
-`kill {{-L|--table}}`
+`kill -l`
```

Comment:
In the bash builtin, `-L` is equivalent to `-l`.
In util-linux, `-l` prints a list without the signal numbers.
```suggestion
`kill -L`
```